src/webgl/loading.js

```diff
@@ -241,6 +275,308 @@ function parseObj(model, lines) {
   return model;
 }
 
+/**
+ * STL files can be of two types, ASCII and Binary,
+ *
+ * We need to convert the arrayBuffer to an array of strings,
+ * to parse it as an ASCII file.
+ */
+function parseSTL(model, buffer) {
+  if (isBinary(buffer)) {
+    parseBinarySTL(model, buffer);
+  } else {
+    var reader = new DataView(buffer);
+
+    if (!('TextDecoder' in window)) {
+      console.error(
```

Comment:
Hi @sanketsingh24 sorry I didn't catch this before but we try not to use `console.error` in this project. I understand that we want the user to know when there model isn't working and that continuing to load it doesn't make sense but it is better if their sketch works as much as possible and this will cause the whole sketch to come to a halt. This is more of a matter of the design, intention, and philosophy of the library. If a user does something like:
```
let myModel;

function preload() {
  myModel = loadModel('myBrokenSTL');
} 

function draw() {
  background(0, 0, 255);
  model(myModel);
}
```
We would want the person who wrote this to see the blue background. Just to give them the feeling that _something_ they are doing is working. Does that make sense?

Reply:
It perfectly makes sense! I will do the requested.

Reply:
@stalgiag Actually `console.error` does not break the sketch, in fact, your code will actually render the blue background with an error message in the console. Maybe you are confused with `throw new error`, which actually stops the execution. Maybe I can change it to `console.warn` if you want?

Reply:
Oh yes but still better to switch to warn just to make it clear that execution will continue.